andela/ah-jumanji-frontend
# #161966981 Add user like articles functionality

#### What does this PR do?
This add user functionality to like articles

#### Description of Task to be completed?
Build out UI components (Like button and View user likes)
Writes the functionality of the app

#### How should this be manually tested?
Log in to the app.
Navigate to any article and view it.
You should be taken to a page that displays thumb up for liking

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#161966981](https://www.pivotaltracker.com/story/show/161966981)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/39841583/51816928-cb907380-22d9-11e9-8774-0ef237d43470.png)
![image](https://user-images.githubusercontent.com/39841583/51816937-d3e8ae80-22d9-11e9-9ac8-4516c65bc2b4.png)
![image](https://user-images.githubusercontent.com/39841583/51896777-4337ce80-23be-11e9-9acd-a1d64e04d5aa.png)

#### Questions:

#### Checklist:
